pkgs/development/tools/godot4/server.nix

```diff
@@ -0,0 +1,18 @@
+{ godot, lib }:
+godot.overrideAttrs (oldAttrs: rec {
+  pname = "godot-server";
+  sconsFlags = "target=release platform=server tools=no";
```

Comment:
You are sure this is not an array?

Reply:
@SuperSandro2000 this file remains unchanged from the original godot package.